app/api/users.py

```
@@ -0,0 +1,200 @@
+) -> UserResponse:
+    """Get current user information."""
+    try:
+        user = await auth_service.get_current_user_from_token(db, token)
+        return UserResponse.model_validate(user)
+    except HTTPException:
+        raise
+    except Exception as e:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Failed to get user: {str(e)}"
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)